patches/chrome-installer-linux-BUILD.gn.patch

```diff
@@ -36,7 +46,20 @@ index fbabd6c3414f5cb69cf4c7ab6ef5f957cb391376..5c42a601d6b101e3caf43203568fb42f
      ]
    }
  
-@@ -386,6 +385,9 @@ group("installer_deps") {
+@@ -357,6 +361,12 @@ group("installer_deps") {
+     "//chrome/browser/resources/media/mei_preload:component",
+     "//sandbox/linux:chrome_sandbox",
+   ]
++  if (brave_chromium_build) {
```

Comment:
same here and also there's no reason to have two deps here. These are both in `brave/build/linux:brave`